.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/pycqa/flake8
+    rev: 3.9.1
+    hooks:
+    -   id: flake8
+-   repo: https://github.com/pycqa/isort
+    rev: 5.8.0
+    hooks:
+    -   id: isort
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+    -   id: black
```

Comment:
what about `mypy`?

Reply:
Unfortunately, mypy was a tad weird. The mirrored version (which was compatible with `pre-commit`) didn't seem to respect our current `mypy.ini` configuration and raised mypy errors on files in `ddtrace.vendor` and `ddtrace.settings`, regardless of my configurations to exclude them. So I've left them out for now and hopefully can include them in a future PR.